process/github-issues.md

```diff
@@ -0,0 +1,50 @@
+# Security Process Using Github Issues (DRAFT)
+This document outlines the process of using Github issues to coordinate security issues of a given project.
+
+## Should you use Github issues?
+ Before submitting any issues, the disclosure should have already engaged in a conversation with the Status Security team about appropriate methods of disclosure for a given project. This should be described in the SECURITY.md file within any given repository. If it is not, then follow the advice of this repository's README.md. If the discloser is still unsure, then please request guidance at security@status.im. 
+
+## Process
+Once the discloser is aware that Github issues are the appropriate methodology of disclosure, the following procedure is **RECOMMENDED****:
+- A github issue **MUST** be created for ever individual assessment discovery
+- a `difficulty` label **MUST** be added to every issue
```

Comment:
Difficulty of... identifying the issue?

Reply:
this should be specified, difficulty of successfully performing the issue at hand.

Reply:
ie. difficulty of exploiting the vulnerability?

Reply:
yes